README.md

```diff
@@ -1,43 +1,40 @@
 # Book Review Site
 
-Use the skills you've learned to build pages of an online book review website.
-
-This is an individual, [stage 1](https://github.com/Ada-Developers-Academy/pedagogy/blob/master/rule-of-three.md) project.
+We will use the skills we've learned to build pages for an online book review website.
 
 ##  Learning Goals:
 
--  Demonstrate an understanding of HTML syntax
--  Demonstrate an understanding of appropriate tag selection for specific content  
--  Demonstrate an ability to link local pages and remote pages
--  Demonstrate the creation of a external stylesheet  
--  Organize a site with [semantic](http://blog.teamtreehouse.com/use-html5-sectioning-elements) HTML
+- Practice structuring content using [semantic](http://blog.teamtreehouse.com/use-html5-sectioning-elements) HTML.
+- Practice linking local pages and remote pages.
+- Practice creating an external stylesheet.
+- Practice selecting elements and applying styles using CSS. 
 
 
 ## Wave 1 - Creating Semantic HTML
 ### The Online Book Review Site
 
 ![Site Layout](images/layout.png "Site Layout")
 
-In this exercise you will create the HTML to go along with content for 3 pages of HTML.  The pages of your site will be the pieces of an online book review site.  You will structure the content given in the content folder with semantic html tags providing meaning to the content.
+In this exercise, we will create the HTML to go along with content for 3 pages of HTML. The pages of our site will be the pieces of an online book review site. We will structure the content given in the content folder with semantic HTML tags providing meaning to the content.
 
 Each page must have a:
 -  Doctype, head, and body of an HTML    
 -  A header/title Section
```

Comment:
```suggestion
-  A heading/title section
```

if we are talking `h1`, then it's a heading, not a header

Reply:
I think we're saying a [header](https://www.w3schools.com/tags/tag_header.asp), so should we change it to just header from header/title?